pkg/cli/prompt/validator.go

```diff
@@ -77,6 +77,17 @@ func ValidateResourceName(input string) error {
 	return errors.New(invalidResourceNameMessage)
 }
 
+// ValidateApplicationName checks if the given string is a valid Application name, and returns an error if it is not.
+// The rules for application name disallows upper case since we use the name to also create a kubernetes namespace for the application.
+func ValidateApplicationName(input string) error {
+	r := regexp.MustCompile("^[a-z]([a-z0-9-]*[a-z0-9])?$")
+	if r.MatchString(input) {
+		return nil
+	}
+
+	return errors.New(invalidResourceNameMessage)
```

Comment:
Is this the right validation message to show?

Reply:
I have the right validation message now.